gm4_metallurgy/data/gm4_metallurgy/functions/item.mcfunction

```diff
@@ -1,8 +1,8 @@
 # @s = an item entity
 # run from main
 
-#restore placed and broken custom skulls
-execute if entity @s[nbt={Item:{id:"minecraft:player_head"}}] run function gm4_metallurgy:repair_skulls
+#change bands that were placed with an outdated format into mundane bands
+execute if entity @s[tag=!gm4_ml_skull_item,nbt={Item:{id:"minecraft:player_head"}},nbt=!{Item:{tag:{gm4_metallurgy:{}}}}] unless data entity @s Item.tag.SkullOwner.Name run function gm4_metallurgy:repair_skulls
```

Comment:
Instead of still doing this, which negates any performance benefits of the player heads library, why not register texture entries for the outdated heads?

Reply:
The reason I did that was because I hadn't thought of checking for `display` instead of `gm4_player_head`.

Reply:
Also, I totally blanked and forgot that the `Name` method doesn't fallback to texture, so I was worried it would break new-format shamirs too.